04-spatial-operations.Rmd

```diff
@@ -520,11 +529,13 @@ plot(z["capacity"])
 <!--### Modifying geometry data; still need to change the corresponding cross-references-->
 
 The result of this join has used a spatial operation to change the attribute data associated with simple features but the geometry associated with each feature has remained unchanged.
+<!-- no longer true, that's why I would remove the sentence
```

Comment:
no longer true (these operations are now in c5), that's why I would remove the sentence here

Reply:
Thanks - I've also tidied up by removing this comment now!